stdlib/asyncio/windows_events.pyi

```diff
@@ -50,10 +50,10 @@ if sys.platform == "win32":
         def recv_into(self, conn: socket.socket, buf: WriteableBuffer, flags: int = ...) -> futures.Future[Any]: ...
         def send(self, conn: socket.socket, buf: WriteableBuffer, flags: int = ...) -> futures.Future[Any]: ...
         def accept(self, listener: socket.socket) -> futures.Future[Any]: ...
-        def connect(self, conn: socket.socket, address: bytes) -> futures.Future[Any]: ...
+        def connect(self, conn: socket.socket, address: tuple[Any, Any] | tuple[Any, Any, Any, Any]) -> futures.Future[Any]: ...
```

Comment:
Use `Incomplete` instead of `Any`? I feel we could tighten this.

Reply:
Yes, this can definitely be more precise.